github-sectory

```diff
@@ -94,14 +94,20 @@ class GithubSectory():
 		puts(colored.yellow("[ Getting folder content ]"))
 		self.downloadLink = parameter
 		try:
-			self.parsed = json.loads(urllib2.urlopen(self.downloadLink).read())
+			self.parsed = json.loads(urllib2.urlopen(self.downloadLink, timeout=30).read())
```

Comment:
I did not include it initially. But I noticed that the process got stuck some times. I found out that if the connection was made and after that the internet connection goes off, then this will leave the socket(which was opened for reading) waiting indefinitely.